### DISCUSSION TITLE
WhatsApp 360 dialog is missing from 3.9

### DISCUSSION THREAD

**Discussion by harismateen** [chatwoot/chatwoot#9554] [Q&A]
Hi, may i check the reason to remove whatsapp 360dialog from 3.9? And how are we going to handle as currently we already have multiple inboxes there....

**Comment by vishnu-narayanan** [comment]
> Going forward, we will be deprecating 360Dialog as a WhatsApp provider. Hence, new inboxes cannot be created with this provider starting from this release. Existing inboxes will continue to work, and a migration path will be provided

3.9 changelog --> https://github.com/chatwoot/chatwoot/releases/tag/v3.9.0

**Reply by harismateen** [reply]
@vishnu-narayanan Thank you for your kind response. May I ask the reason for this deprecation?

The alternative options, such as META Cloud API, are not ideal. META webhooks sometimes trigger automatically, causing duplicate messages, and their APIs are not stable. Additionally, Twilio is very expensive. I would suggest not deprecating 360dialog, as their APIs are more stable and reliable from a financial perspective.

**Reply by harismateen** [reply]
@vishnu-narayanan i would suggest not to remove them as the META APIs and webhooks are not reliable.

**Reply by mattiamarras** [reply]
> > Going forward, we will be deprecating 360Dialog as a WhatsApp provider. Hence, new inboxes cannot be created with this provider starting from this release. Existing inboxes will continue to work, and a migration path will be provided
> 
> 3.9 changelog --> https://github.com/chatwoot/chatwoot/releases/tag/v3.9.0

Is there a migration path? How can I move my number from 360dialog to WhatsApp Cloud? I have already my app up and running on meta dev.
